Source/NSCell.m

```diff
@@ -3021,21 +3078,27 @@ - (void) _drawText: (NSString*)aString  inFrame: (NSRect)cellFrame
 
   attributes = [self _nonAutoreleasedTypingAttributes];
   titleSize = [aString sizeWithAttributes: attributes];
+  if ([self _shouldShortenStringForRect: cellFrame size: titleSize length: [aString length]])
```

Comment:
The shortening of a rendered string using an ellipsis seems to be a new behavior unrelated to XIB5 loading. It's welcome, though. Let's pull this in, but I would usually be happier if it was sent to me as a separate change.

Reply:
Yes, this is unrelated, but comes originally from the Eggplant branch as well. I just rewrote the whole code to understand it better.
Here it would be great if you could help me cherry pick these changes into a separate pull request. I know you can do that :-)

Reply:
> Here it would be great if you could help me cherry pick these changes into a separate pull request. I know you can do that :-)

Yeah, it would likely be doable with `git rebase -i $BASE_COMMIT` or similar (which is interactive even if not a GUI), but that would likely break Github pull request review GUI even worse than what it did up there :(

So either we do it as the very last step, or if you plan on squishing all the commits when 'merging' onto `master`, it's likely easiest just to create a new commit that removes these lines and creates new ones.

I can probably sit down tomorrow evening after work and pull this PR into a local working copy, test it, split this off and send it to you as a new PR onto your branch + a new PR for the new changes. 

Not sure if it's fully worth it, which is why I said (emphasis added):

> **Let's pull this in**, but I would **_usually_ be happier** if it was sent to me as a separate change.

Reply:
OK, so we just leave it in this pull request the way it is.